skeleton/.buildkite/pipeline.yml

```diff
@@ -0,0 +1,36 @@
+steps:
+  - label: Check Hydra evaluation of release.nix
+    commands:
+      - "cd skeleton"  # TODO: remove this line
+      - "nix-build -A iohkLib.check-hydra -o check-hydra.sh"
+      - "./check-hydra.sh"
+    agents:
+      system: x86_64-linux
+
+  # This will ensure that the generated Nix code is kept up to date.
+  - label: Check auto-generated Nix
+    commands:
+      - "export NIX_PATH=iohk_nix=$PWD:$NIX_PATH"  # TODO: remove this line
+      - "cd skeleton"  # TODO: remove this line
+      - "nix-build -A iohkLib.check-nix-tools -o check-nix-tools.sh"
+      - "./check-nix-tools.sh"
```

Comment:
should `check-nix-tools.sh` be in `.gitignore` (or alternatively, should this use ` --no-out-link`)?

Reply:
We are already having trouble with automatic GC running during script execution (see #139).
That's why I want to keep the result links there, so there are GC roots.
Buildkite will clean the project git directory before the next build.